transcom/mymove
# [MB-16288] Display Link to Documents in Requested Service Items

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16288?atlOrigin=eyJpIjoiM2M0YmM3MjBmM2M4NGRjOWIzYjVkYmVkNGY0NjNhYTIiLCJwIjoiaiJ9)

## Summary
-Added "Download service item documentation:" section in the Requested Service Items table with download links to each service item doc.
-Updated Storybook component for RequestedServiceItemsTable.jsx
-Updated tests to include download service item documentation links in ServiceItemDetails.test.jsx

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Log into office app as prime simulator
2. Select a move. Scroll down to "Service Items" and upload some documents.
3. Log in as TOO and find the same move in the queue.
4. Under 'Requested Service Items" section, you should see the download links under "Download service item documentation:"

### Frontend


## Screenshots
![Screenshot 2023-07-03 at 10 25 38 AM](https://github.com/transcom/mymove/assets/110994428/0ba87e50-eb0f-42da-9a01-5707a6ceb31d)
